tests/e2e/psp22_pallet/psp22.tests.ts

```diff
@@ -0,0 +1,82 @@
+// import { consts } from '../constants'
+// import { expect, setupContract } from '../helpers'
+//
+// describe('MY_PSP22', () => {
+//     async function setup() {
+//         return setupContract('my_psp22_pallet', 'new', 123, '0', '1000')
```

Comment:
To deploy a contract on the local node(from https://github.com/paritytech/substrate-contracts-node/pull/146) you need to pass 10000 Units in the constructor. It is required deposit to create and Asset

Reply:
It works with polkadot js and don't work with redspot in integration tests. I'll try tu use polkadot js directly